inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
+- [DualSPHysics](https://github.com/DualSPHysics/DualSPHysics)
+- [OpenFOAM](https://www.openfoam.com/)
+- [SWASH](https://swash.sourceforge.io/)
+- [XBeach](https://oss.deltares.nl/web/xbeach/)
+- [GROMACS](https://www.gromacs.org/)
+
+Check the documentation of each simulator to learn more on how to configure them. Here, we **highlight** how to use the simulators via Inductiva API and how to scale your simulations with simplicity. There is a general structure that all simulators follow, and then there are some specificities for each simulator.
+
+## Simulators via Inductiva API
+
+To run a simulation you will need to have prepared the configuration files for the simulator you desire to use. This configuration files should be in a folder that will be passed to the simulator and your simulation is ready to start running.
+
+Here, we present an example with DualSPHysics - a Smoothed-Particle Hydrodynamics simulator - and show how it looks in practice. To simplify we provide the configuration files for this example.
+
+### Example
+
+This follows a classical example in CFD with flow around a cylinder.
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Download the configuration files into a folder
+input_dir = inductiva.utils.files.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "dualsph-flow-cylinder.zip"
+)
+
+# Initialize the Simulator
+simulator = inductiva.simulators.DualSPHysics()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir)
+```
+
+And that is it! Your simulation is already running on the cloud and you obtain a task object that allows you to manage it. You can check it's status with `task.get_status()`, wait for it to finish with `task.wait()`, and download the results with `task.get_output()`.
+
+With Inductiva API you don't have immediate access to visualization tools of these simulators. However, you can download the results and use the visualization tools of your choice. For example, for DualSPHysics you can use [Paraview](https://www.paraview.org/) to visualize the results.
+
+## DualSPHysics simulator
+
+DualSPHysics is a Smoothed-Particle Hydrodynamics (SPH) simulator. The simulator is usually configured by a single file with the extension `.xml`. This file contains all the information about the simulation, including the geometry, the physical properties of the fluids, the boundary conditions, the numerical parameters, and the output files. Sometimes the configuration can also use extra geometry files. 
+
+The default name used for the configuration file is `config.xml`. However, you can use any name for the configuration file. In this case, you will need to specify the name of the configuration file when running the simulation. For example, if your configuration file is named `my_config.xml` you can run the simulation with:
```

Comment:
maybe you can reduce this explanation to just one line! I think users understand that they need to put the correct name.